### DISCUSSION TITLE
【灌水】Fairy和牛肉谁厉害

### DISCUSSION THREAD

**Discussion by WuLiao000000** [share121/inter-knot#3173] [其他]
![1723912871210](https://github.com/user-attachments/assets/f876d215-52ae-4ccb-adbd-c723df9981bf)

**Comment by HKLHaoBin** [comment]
都很厉害

**Comment by GrunderIndustries** [comment]
不如ZOE(并不
)

**Comment by leftZeap** [comment]
Neruo！因为neruo更会整活（）

**Comment by srsrplk** [comment]
论成本的话，fairy是凭空生成出来的，后续电费贵。neuro则是初期开发成本高，后续费用低。作用上fairy能协助绳匠工作，neuro能直播。建议按需求选择

**Comment by kisakiIMO** [comment]
牛肉heartheartheartheatheartheartwinkwinkwinkwinkwinkwink

**Comment by shawnho1029** [comment]
牛肉可以刀刀烈火，贏!!(放兒歌

**Comment by excellen114514** [comment]
neuro是vtb啊，这一点比fairy强(bushi
论算力和维护成本的话，fairy都更高
不过我还是更喜欢牛肉<img src="https://bbs-static.miyoushe.com/static/2024/07/31/83388b36f5f3e91b1c5815a39b89b15c_8086283022680108242.png" alt="云璃 好吃" width="75" >

**Comment by Starpt33** [comment]
谁算得出来10+9=？谁厉害（）

**Comment by Gwiazda-0** [comment]
9+10=21！！<3
